app/src/main/res/layout/fragment_home.xml

```diff
@@ -3,4 +3,28 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"/>
+    android:layout_height="match_parent" >
+
+    <EditText
+        android:id="@+id/answer"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="60dp"
+        android:ems="10"
+        android:inputType="textPersonName"
+        android:text="Name"
```

Comment:
use android:hint instead of android:text.  hint is more like a label; it goes away when you put your cursor in the component.  Text is treated like text the user entered; the user must physically delete it each time he/she enters that field... which can be a bad user experience.